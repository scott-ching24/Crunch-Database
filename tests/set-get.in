set a 1 2 
set a 2 3 
get a 
set b 2 3 
set b 1 2 
get b
bye
