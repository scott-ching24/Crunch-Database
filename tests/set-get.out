> ok

> ok

> [2 3]

> ok

> ok

> [1 2]

> bye
